static/app/components/acl/featureDisabled.tsx

```diff
@@ -182,18 +156,32 @@ const HelpDescription = styled('div')`
   }
 `;
 
-const AlertWrapper = styled('div')`
-  ${HelpButton} {
-    color: #6d6319;
-    &:hover {
-      color: #88750b;
-    }
+const HelpText = styled('p')`
+  margin-bottom: ${space(1)};
+`;
+
+const ToggleButton = styled(Button)`
+  color: ${p => p.theme.active};
+  height: ${p => p.theme.text.lineHeightBody}em;
+  min-height: ${p => p.theme.text.lineHeightBody}em;
+
+  &:hover {
+    color: ${p => p.theme.activeHover};
   }
 
-  pre,
-  code {
-    background: #fbf7e0;
+  ${ButtonLabel} {
+    display: grid;
+    grid-auto-flow: column;
+    gap: ${space(1)};
   }
 `;
 
+const CopyButton = styled(Button)`
+  margin-left: auto;
+`;
+
+const Pre = styled('pre')<{insideAlert?: boolean}>`
```

Comment:
Why does it need this type `insideAlert` ?

Reply:
Oh right that's from a previous commit, where the `pre` would have a different, light yellow background if it's inside an alert (and light gray if it's outside). This is no longer necessary, so I'll just remove this prop.